core/src/main/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriter.java

```diff
@@ -425,14 +425,8 @@ void forceSorterToSpill() throws IOException {
           long bytesToTransfer = partitionLengthInSpill;
```

Comment:
Minor nit: we don't need this `bytesToTransfer` anymore. We can remove this mutable variable and just use `partitionLengthInSpill` in its place.